usaspending_api/api_contracts/contracts/v2/awards/award_id.md

```diff
@@ -873,7 +876,7 @@ This endpoint returns a list of data that is associated with the award profile p
 + `office_agency_name` (required, string, nullable)
 
 ## ParentDetails (object)
-+ `agency_id` (required, string)
++ `agency_id` (required, number)
```

Comment:
I changed the API contract since the API is already returning an integer here. 
![image](https://user-images.githubusercontent.com/22781949/87977093-60b59800-ca8b-11ea-8f2d-14375d7bad66.png)

Between
1. Changing API to send string
2. Changing Contract to number to match API
Which would you prefer?

Reply:
Gotcha, shouldn't cause any issues on FE then. Option 2 is good!